docs/source/getting-started/tracking-server-overview/index.rst

```diff
@@ -0,0 +1,327 @@
+which is a bit different from the MLflow UI on Databricks CE. Below is a screenshot of the landing page:
+
+.. image:: ../../_static/images/quickstart/tracking-server-overview/mlflow-localhost-landing-page.png
+    :width: 800px
+    :align: center
+    :alt: Landing page of OSS MLflow server
+
+It's also possible to deploy your own MLflow server on cloud platforms, but it is out of the scope of this guide.
+
+
+Connect MLflow Session to Your Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now the server is spinned up, let's connect our MLflow session to the local server. This is very 
+similar to how we connect to Databricks platform.
+
+.. code-block:: python
+
+  mlflow.set_tracking_uri("http://localhost:5000")
+
+
+Then let's try logging some dummy metrics, and view them on the local hosted UI:
+
+.. code-block:: python
+
+  mlflow.set_experiment("/check-localhost-connection")
+
+  with mlflow.start_run():
+      mlflow.log_metric("foo", 1)
+      mlflow.log_metric("bar", 2)
+
+
+Putting it together you can copy the following code to your editor and save it as `test_mlflow_localhost.py`:
+
+.. code-block:: python
+
+  import mlflow
+
+  mlflow.set_tracking_uri("http://localhost:5000")
+  mlflow.set_experiment("check-localhost-connection")
+
+  with mlflow.start_run():
+      mlflow.log_metric("foo", 1)
+      mlflow.log_metric("bar", 2)
+
+
+Then execute it by:
+
+.. code-block:: bash
+
+  $ python test_mlflow_localhost.py
+
+
+View Experiment on Your MLflow Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now let's view your experiment on the local server. Open the URL in your browser, which is `http://localhost:5000` 
+in our case. In the UI, inside the left sidebar you should see the experiment with name 
+`"check-localhost-connection"`, clicking on it should bring you to the experiment view similar as below. 
+
+.. image:: ../../_static/images/quickstart/tracking-server-overview/mlflow-localhost-experiment-view.png
+    :width: 800px
+    :align: center
+    :alt: Experiment view of OSS MLflow server
+
+Clicking on the run (`"clumsy-steed-426"`) will bring you to the run view, similar as below.
+
+.. image:: ../../_static/images/quickstart/tracking-server-overview/mlflow-localhost-run-view.png
+    :width: 800px
+    :align: center
+    :alt: Run view of OSS MLflow server
+
+Conclusion
+^^^^^^^^^^^
+
+That's all about how to start your own MLflow server and view your experiments. Please see the pros and cons
+of this method below:
+
+* **Pros**
+  
+  * You have full control of your data and model, which is good for privacy concerns.
+  * No subscription is required.
+  * Unlimited quota of experiments/runs.
+  * You can even customize your UI by forking the MLflow repo and modify the UI code.
+
+* **Cons**
+  
+  * Requires manual setup and maintenance.
+  * Team collaboration is harder than using a hosted tracking server, e.g., Databricks CE.
```

Comment:
Databricks CE isn't collaborative, though. It's single-user, right? We might not want to push too hard on the "use Databricks" angle here. Can we make this provider agnostic (AzureML has hosted MLflow as well)

Reply:
done